coding-concepts.js

```diff
@@ -40,22 +41,22 @@ const myCodingSkills = {
   databases: "PostgreSQL",
   versionControl: "GitHub"
 }
-// console.log(myCodingSkills.languages[0])
+ //console.log(myCodingSkills.languages[0])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: JavaScript
+// b) Verify and explain: So this one is telling us to look at the key which is languages by using dot notation and then specifically what is at [0] index of that key. which gives the value JavaScript since its at the [0] index of the key languages
 
 // --------------------5) What will this log?
 
 class Learn {
-  constructor(name) {
+  constructor(name, cohort, year) {
     this.student = name
-    this.cohort = "Alpha"
-    this.year = 2024
+    this.cohort = cohort //"Alpha"
+    this.year = year //2024
   }
 }
-const learnStudent = new Learn("George")
-// console.log(learnStudent)
+const learnStudent = new Learn("George","Bravo", 2040)
+console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: {student: George cohort: Alpha year: 2024}
+// b) Verify and explain: so it printed Learn { student: 'George', cohort: 'Alpha', year: 2024 }. i was not fully right becuase it also printed out Learn. from what i remember on classes is when you are setting a variable learnStudent = new Learn this is giving you the option to change what you have in the class but it depends on what you have the parameters set to in the constructor. so since name is the only parameter thats why its only changing this.student. i beleive if you add cohort and year as parameters in the constructor and then set the value in the keys to cohort and year. you would be able to add onto line 59 to change it all. im going to try it. Output for that is //Learn { student: 'George', cohort: 'Bravo', year: 2040 }
```

Comment:
You were basically correct.  The console log will always include the Class name, but you got the object correct(minus commas and quotes 😃).  You have the concepts down.  You're rocking this assessment!